### PR TITLE
feat(service/sqld): init sqld service

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

**Comment by rawkode** [comment]
Someone should have run the tests locally ... 😅

File: src/modules/services/sqld.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      type = with lib.types; listOf str;
+      default = [ ];
+      description = "Add other sqld flags.";
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    processes.httpbin = {
+      exec = ''
+        exec ${pkgs.sqld}/bin/sqld --http-listen-addr 127.0.0.1:${cfg.port} ${qs cfg.extraArgs}
```
Comment (domenkozar) [inline src/modules/services/sqld.nix:31]:
```suggestion
        exec ${pkgs.sqld}/bin/sqld --http-listen-addr 127.0.0.1:${toString cfg.port} ${qs cfg.extraArgs}
```

File: src/modules/services/sqld.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    extraArgs = lib.mkOption {
+      type = with lib.types; listOf str;
+      default = [ ];
+      description = "Add other sqld flags.";
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    processes.httpbin = {
```
Comment (domenkozar) [inline src/modules/services/sqld.nix:29]:
```suggestion
    processes.sqld = {
```

File: tests/sqld/.test.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+set -e
```
Comment (domenkozar) [inline tests/sqld/.test.sh:1]: ./.test.sh: Permission denied

You can also just inline it into `enterTest`

**Comment by rawkode** [comment]
I've fixed the `+x`

**Pull request merged**